webhooks/elbv2/targetgroupbinding_mutator_test.go

```diff
@@ -41,7 +41,7 @@ func Test_targetGroupBindingMutator_MutateCreate(t *testing.T) {
 		wantErr error
 	}{
 		{
-			name: "targetGroupBinding with TargetType and ipAddressType already set",
+			name: "targetGroupBinding with TargetType and ipAddressType and vpcId already set",
```

Comment:
Can we have a separate unit test where the `vpcId` is not set?

Reply:
comment addressed